afids_utils/features/regRF.py

```diff
@@ -0,0 +1,61 @@
+"""Core methods / functions for regRF"""
```

Comment:
I would rather this module (and `test_regRF`) be called `reg_rf` in keeping with the PEP 8 module name convention